ui_framework/src/components/popover/_popover.scss

```diff
@@ -10,7 +11,7 @@
       opacity: 1;
       visibility: visible;
       z-index: 2;
```

Comment:
This was a mistake of mine. You should use a variable for this... `$kuiZContent`